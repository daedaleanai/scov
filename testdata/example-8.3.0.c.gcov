version:8.3.0
file:example.c
function:28,59,1,main
lcount:28,1,0
lcount:34,1,0
lcount:36,1,0
lcount:37,1,0
lcount:43,1,0
lcount:44,1,0
lcount:49,1,0
lcount:51,0,1
lcount:55,1,0
lcount:58,1,0
