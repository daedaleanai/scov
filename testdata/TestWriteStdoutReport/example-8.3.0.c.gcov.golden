  Line coverage: [##################  ]  90.0%  (9/10)
  Func coverage: [####################] 100.0%  (1/1)
Branch coverage:  No data
Region coverage:  No data
